### ISSUE TITLE
Add Global Privacy Control branding

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/optmeowt-browser-extension#126]
I think we should add some Global Privacy Control (GPC) branding to OptMeowt. Maybe, the GPC logo at the bottom of the popup. Feel free to go ahead if anyone wants to ...

**Comment by kalicki1** [comment]
I added the main Global Privacy Control logo to OptMeowt at the bottom of the popup in the last commit to the main branch. I figured it was a small enough change we could just pop it right in there. Here's how it looks: 

<img width="345" alt="Screen Shot 2021-01-04 at 1 07 09 AM" src="https://user-images.githubusercontent.com/60238705/103505821-3123fd80-4e29-11eb-8f36-5af7831e6e92.png">


We could also drop in some more branding on the settings page, but let's discuss this in tomorrow's meeting.
